src/SoundList.js

```diff
@@ -9,7 +9,35 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+// Psuedo code:
+// function highlightSound(track, sound) {
+//   let packSoundArr = pack sounds array;
+//   let similarSoundArr = similar array;
+//   for (i = 0; i<=packSoundArr.length; i++) {
+//     for (j=0; j <= similarSoundArr; j++) {
+//       if (track.name === sound.name) {
+//         keep background color set to secondary color regardless of hover class
+//       }
+//     }
+//   }
+// }
+
+const SoundList = ({
+  tracks,
+  header,
+  onSoundClick,
+  currentTrackId,
+  className = "",
+}) => {
+  const selectedTrackClassName =
```

Comment:
Threre's a good amount of duplication between these two classes. I recommend the following approach:

```tsx
const Sound = ({isActive}) => {
  const selectedStyles = isActive ? 'my-primary-class my-primary-text' : 'my-secondary-class my-secondary-text';
  return (
    <div className={`some-class-a some-class-b ${selectedStyles}`} />
  )
}
```

Reply:
Ah, I thought there was a cleaner approach. I'll work on the implementation! Do we want the current sound not showing up in the Similar sound list to be part of this pull request?

Reply:
>  Do we want the current sound not showing up in the Similar sound list to be part of this pull request?

Yea I think that would be okay!